src/routes/(console)/account/organizations/+page.svelte

```diff
@@ -36,6 +36,27 @@
         return memberships.memberships.map((team) => team.userName || team.userEmail);
     }
 
+    async function getPlanName(billingPlan: string | undefined): Promise<string> {
+        if (!billingPlan) return 'Unknown';
+
+        // For known plans, use tierToPlan
+        const tierData = tierToPlan(billingPlan as Tier);
+
+        // If it's not a custom plan or we got a non-custom result, return the name
+        if (tierData.name !== 'Custom') {
+            return tierData.name;
+        }
+
+        // For custom plans, fetch from API
+        try {
+            const plan = await sdk.forConsole.billing.getPlan(billingPlan);
```

Comment:
we can do a quick cached data lookup first to avoid a network call, there should be a `plansInfo` available.

Reply:
@ItzNotABug `tierToPlan` essentially has the same data as `plansInfo`, right?